GNUmakefile

```diff
@@ -140,7 +140,7 @@ operator-ui: ## Fetch the frontend
 	go run operator_ui/install.go .
 
 .PHONY: generate
-generate: codecgen mockery protoc gomods ## Execute all go:generate commands.
+generate: codecgen mockery protoc gomods modgraph ## Execute all go:generate commands.
```

Comment:
There is already a `//go:generate make modgraph` in `main.go`. This seems cleaner, but I don't recall why I didn't it that way :thinking: